### PR TITLE
Added build script for nltk develop

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: n/nltk/nltk_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+if ! tox -e py38; then
+	echo "------------------Build_Test_fails---------------------"
+	exit 1
+else
+	echo "------------------Build_Test_success-------------------------"
+	echo "$PACKAGE_NAME  | $PACKAGE_VERSION |"
+fi
+
+#Test passes on VM
+#Does not pass travis check due to sudo command not found.
```
Comment (seth-priya) [inline n/nltk/nltk_ubi_8.5.sh:45]: @Siddhesh-Ghadi @kandarpamalipeddi   is there any way to handle this other than setting Travis-Check to false?

File: n/nltk/nltk_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+#Build and test logs:
```
Comment (seth-priya) [inline n/nltk/nltk_ubi_8.5.sh:47]: @MuskaanSheik I am not sure we want the full logs here

File: n/nltk/nltk_ubi_8.5.sh (hunk 1 above)
Comment (seth-priya) [reply n/nltk/nltk_ubi_8.5.sh:45]: @MuskaanSheik as discussed please install sudo in the script and see if that works

File: n/nltk/nltk_ubi_8.5.sh (hunk 2 above)
Comment (Siddhesh-Ghadi) [reply n/nltk/nltk_ubi_8.5.sh:47]: You can attach the log tail in the PR comment so that reviewer can view it there.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**